man/udev.xml

```diff
@@ -514,6 +514,11 @@
             <term><varname>LABEL</varname></term>
             <listitem>
               <para>A named label to which a <varname>GOTO</varname> may jump.</para>
+
+              <para>Although <varname>LABEL</varname> keys are assignments,
+              they are processed before all other keys. In each rule only the
+              last <varname>LABEL</varname> key takes effect, all the rest are
+              ignored.</para>
```

Comment:
here, as in the GOTO case I'd just say "multiple LABEL assignments within the same line are not supported".

let's not make people rely on bad ideas just because we documented them and gave them guarantees this would work that way...

Reply:
OK